odh/overlays/crc/monitoring/example-github-secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
this secret you can add in the kustomization, since it doesn't require sops, be sure to add the namespace field to the kustomization (otherwise the kustomize build won't add a namespace field to the secret, we'll be doing this to all overlays as part of another issue as well, but you can do it for just the monitoring folder in crc/quicklab, see #99 ) 

you can do the same for the quicklab secret

Reply:
ack, updated :+1: